File: server/freeScoreBoard7/Core/DB/TKopf.cs
namespace FreeScoreBoard.Core.DB
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("TKopf")]
    public partial class TKopf
    {
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long ID { get; set; }

        public int TurnierNr { get; set; }

        [StringLength(2147483647)]
        public string Beschreibung { get; set; }

        [StringLength(2147483647)]
        public string Liga { get; set; }

        [StringLength(2147483647)]
        public string Kommentar { get; set; }

        [StringLength(2147483647)]
        public string Matrix { get; set; }

        [StringLength(2147483647)]
        public string Platzierungstyp { get; set; }

        [StringLength(2147483647)]
        public string Vorkommnisse { get; set; }

        public int Mananz { get; set; }

        public int status { get; set; }

        [StringLength(2147483647)]
        public string Version { get; set; }

        [StringLength(2147483647)]
        public string Turniertyp { get; set; }

        public int MatrixID { get; set; }
    }
}

File: control page/src/turniere/dataClasses/TKopf.js
export default class TKopf {
  constructor(
    _ID,
    _TurnierNr,
    _Beschreibung,
    _Liga,
    _Kommentar,
    _Matrix,
    _Platzierungstyp,
    _Vorkommnisse,
    _Mananz,
    _status,
    _Version,
    _Turniertyp,
    _MatrixID,
  ) {
    this.ID = _ID;
    this.TurnierNr = _TurnierNr;
    this.Beschreibung = _Beschreibung;
    this.Liga = _Liga;
    this.Kommentar = _Kommentar;
    this.Matrix = _Matrix;
    this.Platzierungstyp = _Platzierungstyp;
    this.Vorkommnisse = _Vorkommnisse;
    this.Mananz = _Mananz;
    this.status = _status;
    this.Version = _Version;
    this.Turniertyp = _Turniertyp;
    this.MatrixID = _MatrixID;
  }
}
